### PR TITLE
zh-translation:/faq/metrics-and-logs/life-of-a-request.md

### PR DESCRIPTION
**Pull request by luxious**
ref: https://github.com/servicemesher/istio-official-translation/issues/1030

### CODE REVIEW

File: content/zh/faq/metrics-and-logs/life-of-a-request.md [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
+    # 列出所有的 virtual services
     $ kubectl get virtualservices
     {{< /text >}}
 
-* Mixer access logs: Mixer writes access logs that contain information about requests. You can get them with:
+* Mixer 访问日志： Mixer 的访问日志中包含了关于请求的信息。您可以通过这样获取：
 
     {{< text plain >}}
-    # Fill <istio namespace> with the namespace of your istio mesh. Ex: istio-system
+    # 在 <istio namespace> 填入您自己的 Istio namespace。 比如： istio-system
```
Comment (yuxiaobo96) [inline content/zh/faq/metrics-and-logs/life-of-a-request.md]: 翻译不是很准确

Comment (luxious) [reply content/zh/faq/metrics-and-logs/life-of-a-request.md]: 将 <istio namespace> 处改为您自己的 Istio namespace

这样如何

**Comment by tanjunchen** [comment]
please fix conclicts.Thanks

**Pull request merged**